README.md

```diff
@@ -234,3 +229,56 @@ where `<your-mattermost-url>`, `<your-port>`, `<your-secret>`, `<your-team-url>`
 You can generate a new secret in **System Console > Plugins > Jira**, and paste the new webhook URL in your JIRA webhook configuration. 
 
 This might result in downtime of the JIRA plugin, but it should only be a few minutes at most.
+
+### What changed in the Jira 2.1 Webhook configuration?
+
+In Jira 2.1 there is a modal window for a "Channel Subscription" to Jira issues.  This requires a firehose of events to be sent from Jira to Mattermost and the Jira plugin then "routes" or "drops" the events to particular channels.  The Channel Subscription modal (which you can access by going to a particular channel, then typing `jira /subscribe`) provides easy access for Mattermost Channel Admins to setup which notifications they want to receive per channel.
```

Comment:
0/5
> there is a modal window ~~for a "Channel Subscription"~~ **used to configure "Channel Subscriptions"** to Jira issues

Comma after "from Jira to Mattermost"

I would change the end of the second sentence to just be:
> then routes the events to particular channels based on your subscriptions.